src/Deprecated80/OSPlatform.extension.st

```diff
@@ -0,0 +1,8 @@
+Extension { #name : #OSPlatform }
+
+{ #category : #'*Deprecated80' }
+OSPlatform >> openNativeBrowserOn: aFileReference [ 
+	self deprecated: 'Use message openOSFileBrowserOn:'.
```

Comment:
```suggestion
	self deprecated: 'Use message openOSFileBrowserOn:' transformWith: '`@receiver openNativeBrowserOn: `@statements1' -> '`@receiver v `@statements1'.
```

Reply:
Great. I did not know about transformation rule.